.github/workflows/patch-release.yml

```diff
@@ -0,0 +1,94 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+
+name: Patch Release
```

Comment:
I think it is a good idea to have a yaml file for this than doing steps manually so that's cool - but I would rename the file / naming of this to `release-v0.14.0.yml`, etc. to prevent confusion in the future. This isn't a general "patch release" workflow, as usually we would build a new binary. This is a special workflow only to update ECS configs for this specific release and we will want to delete it right away after - a normal patch release workflow is for building a binary with cherrypicks into a release branch, something like https://github.com/open-telemetry/opentelemetry-java/blob/main/.github/workflows/patch-release-build.yml